### DISCUSSION TITLE
Where is the source code of command-not-found.com?

### DISCUSSION THREAD

**Discussion by SteveLauC** [tldr-pages/tldr#11793] [Q&A] [answered]
Hi there, I would like to know where the source code of that [command-not-found.com](https://command-not-found.com/) is as I want to fix the following mistake:

The instruction on how to install `perf` on Debian is wrong, it should be :

```sh
$ apt-get install linux-perf
```

rather than:

```sh
$ apt-get install perf
```

**Comment by kbdharun** [comment, marked as answer]
Hi, We don't provide installation instructions for commands on our pages and that section of [command-not-found.com](https://command-not-found.com/) isn't affiliated with our project. I too am not sure about where the source code is, so I would suggest directly mailing the author of the website regarding this issue at <lukasz@lach.dev> or some other listed contact methods at <https://linktr.ee/lach.dev>.

**Reply by SteveLauC** [reply]
Thanks!
